client/src/settings/manage.tsx

```diff
@@ -69,15 +69,15 @@ export function Manage() {
         )}
 
         <li>
-          <h3>Firefox account</h3>
-          <span>Manage your Firefox account</span>
+          <h3>Your account</h3>
```

Comment:
Why don't we just call it "Account"? We don't call it "Your subscription" either.

```suggestion
          <h3>Account</h3>
```

cc @s-sood

Reply:
Let's just do "Account"